clusterloader2/pkg/config/template_functions.go

```diff
@@ -187,6 +188,15 @@ func isEven(number interface{}) bool {
 	return int(toFloat64(number))%2 == 0
 }
 
+func isModIn(a interface{}, mod interface{}, acceptedValues ...interface{}) bool {
```

Comment:
Since you introduce it, can you also remove IsEven and IsOdd ?

Reply:
Will do that in a follow up PR, I have some other things to change as well.